src/Sfa.Tl.ResultsAndCertification.Api.Client/Clients/DfeSignInApiClient.cs

```diff
@@ -39,27 +42,32 @@ public async Task<DfeUserInfo> GetDfeSignInUserInfo(string organisationId, strin
 
             var userInfoResult = userInfo.Result;
             var ukprn = organisationUkprn.Result;
+            userInfoResult.Organisation = organisationUkprn.Result.Name;
 
-            if(ukprn.HasValue)
-                userInfoResult.Ukprn = ukprn;
+            if (ukprn.UKPRN.HasValue)
+                userInfoResult.Ukprn = ukprn.UKPRN;
+            else if (!ukprn.UKPRN.HasValue) userInfoResult.HasAccessToService = HasAccesstoService(Common.Constants.OrganisationConstants.AdminOrganisation, userInfoResult, RolesExtensions.AdminDashboardAccess);
             else
                 userInfoResult.HasAccessToService = false;
             
             return userInfoResult;
         }
 
-        private async Task<long?> GetOrganisationUkprn(string organisationId, string userId)
+        private async Task<Organisation> GetOrganisationUkprn(string organisationId, string userId)
```

Comment:
This returns an Organisation, not a UKPRN. It should be named GetOrganisation

Reply:
Done the change.